specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -524,6 +629,86 @@
           "minimum": 1,
           "maximum": 102400,
           "description": "The maximum size of the share, in gigabytes. Must be greater than 0, and less than or equal to 5TB (5120). For Large File Shares, the maximum size is 102400."
+        },
+        "enabledProtocols": {
+          "type": "string",
+          "enum": [
+            "SMB",
+            "NFS"
+          ],
+          "x-ms-enum": {
+            "name": "EnabledProtocols",
+            "modelAsString": true
+          },
+          "description": "The authentication protocol that is used for the file share. Can only be specified when creating a share.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "rootSquash": {
+          "type": "string",
+          "enum": [
+            "NoRootSquash",
+            "RootSquash",
+            "AllSquash"
+          ],
+          "x-ms-enum": {
+            "name": "RootSquashType",
+            "modelAsString": true
+          },
+          "description": "The property is for NFS share only. The default is NoRootSquash."
+        },
+        "version": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The version of the share."
+        },
+        "deleted": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Indicates whether the share was deleted."
+        },
+        "deletedTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The deleted time if the share was deleted."
+        },
+        "remainingRetentionDays": {
+          "type": "integer",
+          "readOnly": true,
+          "description": "Remaining retention days for share that was soft deleted."
+        },
+        "accessTier": {
```

Comment:
What's the value user can set when update/create share?
I think not all value can set set by user, like user should can't set it to premium from other value.
Would you please add it to description to make the usage more clear?

Reply:
Edited the description.

Reply:
hot and cool can only be set to GPV2, or to all accounts?

Reply:
GPv2 only. We don't support GPv1. FileStorage is Premium only.

Reply:
Would you please indicate it in the description?